src/components/Players.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { getPlayers } from '../actions/actions.js';
+import { selectPlayer } from '../actions/actions.js';
+import { getPlayerHeadshot } from '../actions/actions.js';
+import { getPlayerDetails } from '../actions/actions.js';
+
+class Players extends React.Component {
+    componentDidMount() {
+
+        this.props.getPlayers();
+        // this.props.getPlayerDetails();
+        console.log('this.props.getPlayers()', this.props.getPlayers());
```

Comment:
Always remove console.log in PR. Unless, you need to leave a console, in those very few cases, use `console.info('this.props.getPlayers()', this.props.getPlayers());`

I'll let you decide between one of these choices